tests/pytorch_tests/model_tests/feature_models/qat_test.py

```diff
@@ -134,3 +138,50 @@ def compare(self, ptq_model, qat_ready_model, qat_finalized_model, input_x=None,
             # check quantization didn't change when switching between PTQ model and QAT ready model
             qat_finalized_output = qat_finalized_model(_in).cpu().detach().numpy()
             self.unit_test.assertTrue(np.isclose(np.linalg.norm(qat_finalized_output - qat_ready_output) / np.linalg.norm(qat_ready_output), 0, atol=1e-6))
+
+
+class QuantizationAwareTrainingMixedPrecisionCfgTest(QuantizationAwareTrainingTest):
+    def __init__(self, unit_test, weight_bits=2, activation_bits=4, finalize=False,
+                 weights_quantization_method=mct.target_platform.QuantizationMethod.POWER_OF_TWO, test_loading=False):
+
+        self.weight_bits = weight_bits
+        self.activation_bits = activation_bits
+        self.finalize = finalize
+        self.weights_quantization_method = weights_quantization_method
+        self.test_loading = test_loading
+        super().__init__(unit_test)
+
+    def get_tpc(self):
+        base_config, _ = get_op_quantization_configs()
+        return get_mp_activation_pytorch_tpc_dict(
+            tpc_model=generate_tp_model_with_activation_mp(
+                base_cfg=base_config,
+                mp_bitwidth_candidates_list=[(8, 8), (8, 4), (8, 2),
+                                             (4, 8), (4, 4), (4, 2),
+                                             (2, 8), (2, 4), (2, 2)]),
+            test_name='qat_test',
+            tpc_name='qat_test_tpc')['qat_test']
+
+    def run_test(self, experimental_facade=False):
+        model_float = self.create_networks()
+        config = mct.CoreConfig(mixed_precision_config=MixedPrecisionQuantizationConfigV2())
+        num_of_parameters = sum(p.numel() for p in model_float.parameters() if p.requires_grad)
+        # kpi = mct.KPI(num_of_parameters * 0.75)
+        kpi = mct.KPI()
```

Comment:
I think it is important to also have a test with some constrained KPI. Consider adding one.